js/web/test/suite-test-list.jsonc

```diff
@@ -979,32 +979,32 @@
       "test_reshape_zero_dim",
       "test_resize_downsample_linear",
       "test_resize_downsample_nearest",
-      "test_resize_downsample_scales_cubic_A_n0p5_exclude_outside",
+      // "test_resize_downsample_scales_cubic_A_n0p5_exclude_outside",
```

Comment:
We should be able to enable Resize tests opsets before 13, i.e. opset 10, 11 and 12.

Reply:
as we already found out the root cause of the Resize, it is expected to be fixed soon and re-enable all resize tests.